libs/application/templates/accident-notification/src/assets/WorkTypeIllustration.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { FC } from 'react'
+
+export const WorkTypeIllustration: FC = () => (
```

Comment:
We should probably avoid unnecessary use of RC https://github.com/island-is/island.is/pull/3621#discussion_r620690786